templates/master/00-master/gcp/units/openshift-gcp-routes.service.yaml

```diff
@@ -5,7 +5,8 @@ contents: |
   Description=Update GCP routes for forwarded IPs.
   ConditionKernelCommandLine=|ignition.platform.id=gce
   ConditionKernelCommandLine=|ignition.platform.id=gcp
-  After=network.target
+  Wants=network-online.target
+  After=network-online.target
```

Comment:
triple checking on this service - @squeed you wrote the original https://github.com/openshift/machine-config-operator/blob/release-4.5/templates/master/00-master/gcp/units/openshift-gcp-routes.service#L8 w/o the wants but it seems Tim found an issue requiring it for gcp+ovn - just triple checking as this service caused some weird issues in the past

Reply:
as a follow up, looks like Casey is on PTO also - talked to Stefan, we're not fully sure but Tim pointed out this is needed anyway in OVN+GCP

Reply:
Yes from what I found the gcp routes thing relies on network connectivity and finding an interface to use. With these changes that interface will change sometime after network.target but before network-online.target...meaning there could be a race where gcp routes uses the wrong interface. I see GCP just passed with this patch on

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_ovn-kubernetes/216/pull-ci-openshift-ovn-kubernetes-master-e2e-gcp-ovn/1288509043568021504

So I think we are OK.